src/assets/Enums.ts

```diff
@@ -41,3 +42,218 @@ export enum VEHICLE_TYPE {
     TRL = "trl",
     PSV = "psv"
 }
+
+export const vehicleType: string[] = [
+    "psv",
+    "trl",
+    "hgv"
+];
+
+export const fuelPropulsionSystem: string[] = [
+    "DieselPetrol",
+    "Hybrid",
+    "Electric",
+    "CNG",
+    "Fuel cell",
+    "LNG",
+    "Other"
+];
+
+export const vehicleClassDescription: string[] = [
```

Comment:
Naming Convention for all constants is Caps and Underscore. Helps in identifying the constants when used.

Reply:
Please update all cases in the file.

Reply:
done